src/de.rs

```diff
@@ -50,11 +54,13 @@ impl<'b> Deserializer<'b> {
         Ok(value)
     }
 
-    fn read_varint_string(&mut self) -> Result<String> {
-        let string_length = self.read_varint()?;
-        let string = self.read_string(string_length)?;
-
-        Ok(string)
+    fn read_varint_marked_string(&mut self) -> Result<Vec<u8>> {
```

Comment:
Why does this say `string` when it returns `Vec<u8>`?

Reply:
This function decodes the epee field which is marked as a string but yeah that is a bit confusing so I'll change it